coderd/prebuilds/preset_snapshot_test.go

```diff
@@ -708,6 +711,29 @@ func TestExpiredPrebuilds(t *testing.T) {
 					},
 				}
 
+				validateState(t, expectedState, state)
+				validateActions(t, expectedActions, actions)
+			},
+		},
+		{
```

Comment:
Should we maybe also add a test where one of the prebuilds was created after the last_invalidated_at timestamp?